seed/views/v3/organizations_users.py

```diff
@@ -0,0 +1,150 @@
+# encoding: utf-8
```

Comment:
Oh, also, do we keep the first word singular or plural - organization or organization**s**? I'm sure it's completely up to us, so let me know your thoughts.

Reply:
After discussing, singular then plural in the name is 👌

Reply:
After discussing, we decided to just go with singular-plural pattern for filenames